### ISSUE TITLE
Page warning needs to be updated for Windows 10

### ISSUE THREAD

**Issue by ReedKimble** [ARMmbed/Handbook#1493]
# Docs issue template

## Page link
https://os.mbed.com/docs/mbed-os/v6.15/program-setup/windows-serial-driver.html

## Expected to find...
The driver necessary for Windows 10 to recognize an old mBed as both removable media and a COM port.

## Found
A warning not to use this driver for anything later than Windows 7... but it is necessary for Windows 10 in at least some scenarios.

**Comment by 0xc0170** [comment]
Did you need to install the driver for old mbed board on Windows 10?

**Comment by ReedKimble** [comment]
Yes, it was an original mbed board and I had to install that driver for the latest Win10 to recognize everything properly.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

________________________________
